### PR TITLE
[SDPA-5887]updating tide_migration version to D9

### PR DESCRIPTION
**Pull request by yeniatencio**
https://digital-engagement.atlassian.net/browse/SDPA-5887

Update tide_migration to Drupal 9.x

No deprecated functions was found.

### CODE REVIEW

File: tide_migration.info.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: Tide Migrations
 type: module
 description: Migrate content from www.vic.gov.au
-core: 8.x
+core_version_requirement: ^8.9 || ^9
 version: 8.x-1.0
```
Comment (vincent-gao) [inline tide_migration.info.yml]:
```suggestion
```

Comment (vincent-gao) [reply tide_migration.info.yml]: `version: 8.x-10` should be removed.

**Review by vincent-gao — CHANGES_REQUESTED** [review]

File: tide_migration.info.yml (hunk 1 above)
Comment (yeniatencio) [reply tide_migration.info.yml]: Done. Thanks

**Review by vincent-gao — APPROVED** [review]

**Review by FleetAdmiralButter — APPROVED** [review]
There's no CI on this project, can you test this on a D9 project that uses this and make sure it still bootstraps?

**Comment by yeniatencio** [comment]
@FleetAdmiralButter , only content-together uses this module and it is not updated to D9 yet.

**Pull request merged**